bydbctl/README.md

```diff
@@ -5,5 +5,8 @@
 ## Build
 
 ```
+# If you haven't generated the API code yet
+make -C ../api generate
+
```

Comment:
Yes, by changing the Makefile to add prerequirement, something like

```makefile
.PHONY: generate-api
generate-api:
	make -C ../api generate

build: generate-api
```

I've also considered this, but if the directory structure changed this may cause issue

Reply:
Then CI fails.